Makefile

```diff
@@ -2,13 +2,16 @@ SCHEMATA=	DGC.schema.json \
 		DGC.Core.Types.schema.json \
 		DGC.Types.schema.json \
 		DGC.ValueSets.schema.json
-
 MERGED=		DGC-all-schemas-combined.json
 
-AJV=		./node_modules/.bin/ajv -c ajv-formats --spec=draft2020
+JSON_FILES=	*.json examples/*.json valuesets/*.json
+
+AJV=		./node_modules/.bin/ajv -c ajv-formats --spec=draft2020 --strict=false
```

Comment:
You have spec draft 2020
I went for draft-07 as lowest common denominator latest standard
So preferably you have spec=draft07 and strict=true

Reply:
We decided in #13 to use 2020-12. If people want to use earlier draft versions, they can downgrade easily.